frontend/public/components/operator-lifecycle-manager/descriptors/spec/index.tsx

```diff
@@ -30,18 +34,53 @@ const ResourceRequirements: React.SFC<SpecCapabilityProps> = ({obj, descriptor})
   <dd><ResourceRequirementsModalLink type="requests" obj={obj} path={descriptor.path} /></dd>
 </dl>;
 
-const K8sResourceLink: React.SFC<SpecCapabilityProps> = (props) => <ResourceLink kind={props.capability.split(SpecCapability.k8sResourcePrefix)[1]} name={props.value} namespace={props.namespace} title={props.value} />;
+const K8sResourceLink: React.SFC<SpecCapabilityProps> = (props) => _.isEmpty(props.value)
+  ? <span className="text-muted">None</span>
+  : <ResourceLink kind={props.capability.split(SpecCapability.k8sResourcePrefix)[1]} name={props.value} namespace={props.namespace} title={props.value} />;
```

Comment:
I'd remove the `title` attribute here if it's just repeating the name (unless we expect name to get trunacted)